Yosemite/Yosemite/Stores/ShippingLabelStore.swift

```diff
@@ -135,6 +175,17 @@ private extension ShippingLabelStore {
         }
     }
 
+    func handleShippingLabelRefund(refund: ShippingLabelRefund?, storageShippingLabel: StorageShippingLabel) {
```

Comment:
⚠️ It'd be nice if we can rename this to something that is not “handle“. 🙂 If I understand it correctly, the goal is to _update the `refund`_ of `Storage.ShippingLabel`. So maybe something like 

```swift
func update(shippinglabel: ShippingLabel, withRefund refund: ShippingLabelRefund) { }
```

It's not the best but I think it's better than `handle` so future developers will have a better idea of the intent and not blow this method up. 😬

Reply:
Good idea! I was following the naming like how we [update relationships of a `Product` in `ProductStore`](https://github.com/woocommerce/woocommerce-ios/blob/fb2b7595935cd1e38f70630dd8ad0ef5145f3aa9/Yosemite/Yosemite/Stores/ProductStore.swift#L393), but this `update` function sounds much more clear. Updated in: https://github.com/woocommerce/woocommerce-ios/pull/3294/commits/bc7ab8b7b185ef6ba7b3ec0e0563479fecdb4669